Contest_API.md

```diff
@@ -689,13 +689,13 @@ Properties of contest objects:
 | :--------------------------- | :-------------- | :----------
 | id                           | ID              | Identifier of the current contest.
 | name                         | string          | Short display name of the contest.
-| formal\_name                 | string          | Full name of the contest.
+| formal\_name                 | string ?        | Full name of the contest. Defaults to value of `name`.
```

Comment:
I'd still argue that this doesn't have to be nullable, since there's only a single instance of this in the contest object, and thus the server can simply indicate in `/access` whether this will be available. But I can live with this.

Reply:
Ditto, I'd be ok with this not being nullable.